Xamarin/DrawXShared/ViewControllerShared.cs

```diff
@@ -0,0 +1,183 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+using System;
+using System.Diagnostics;
+using DrawXShared;
+using Foundation;
+using SkiaSharp.Views.iOS;
+using UIKit;
+
+namespace DrawX.IOS
+{
+    // Most of the ViewController logic is factored out here so we can subclass
+    // with a local copy in case we want a debug build which bypasses nuget as a Realm source
```

Comment:
I felt a need to explain why the VC was factored into two classes. I don't want to combine them in case doing debugging with a local variant again.